packages/wonder-blocks-tokens/src/tokens/semantic-color.ts

```diff
@@ -85,10 +91,12 @@ export const semanticColor = {
                     foreground: color.activeRed,
                 },
             },
-        },
-        disabled: {
-            default: color.fadedOffBlack32,
-            secondary: color.offWhite,
```

Comment:
Do we need to mark the changeset as a `major` change since we changed the structure for the semantic tokens?